authors/tests/test_likes_dislikes.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework.reverse import reverse
+from rest_framework.views import status
+
+# local imports
+from .base_test import TestBase
+
+
+class TestLikesDislikes(TestBase):
+    """This class has tests for like and dislike test case"""
+
+    def test_like(self):
```

Comment:
Kindly add docstrings for your test cases. I think it will help other people understand your tests betteer.